src/EmployeeRole.src.json

```diff
@@ -0,0 +1,59 @@
+{
+    "type": "EmployeeRole",
+    "uri": "http://schema4i.org/EmployeeRole",
+    "description": "A EmployeeRole describes employee relationship",
+    "links": [{
+        "url": "http://schema.org/EmployeeRole",
+        "description": "Original Schema.org type"
+    }],
+    "parents": [
+        { "@id": "http://schema4i.org/Person#HasOccupation" }
+    ],
+    "base": [
+        { "@id": "http://schema4i.org/Thing" }
+    ],
+    "multipletypes": {},
+    "context": {
+        "@context": {
+            "@version": 1.1,
+            "s4i": "http://schema4i.org/",
+            "schema": "http://schema.org/",
+            "EmployeeRole": "schema:EmployeeRole",
+            "HasOccupation": {
```

Comment:
This is ok, because in S4i we don't know something exactly like [Intangible](https://schema.org/Intangible) in schema.org. But please add an example to show the exact usage in the context of a `Person` -> `HasOccupation` -> `EmployeeRole` -> `HasOccupation` -> `Occupation`.